applications/materialDataCreator/materialDataCreator.html

```diff
@@ -0,0 +1,573 @@
+                </tr></table>
+            
+            </div>
+            
+            <div class='info'>
+            <table style= 'width: 100%;'><tr><td>Name:</td><td style= 'width: 100%;'><input name = 'name' type = 'text' style= 'width: 100%;' oninput="save('name', this);"></td></tr></table>
+            </div>
+            
+            <div class='group'>
+            <font class='label'>Albedo (Color):</font> <input name = 'albedo' type = 'text' value="#ffffff"><br>
+            <font class='label'>Albedo Map (RGB) URL:</font> <input name = 'albedoMap' type = 'text' style= 'width:100%;' oninput="save('albedoMap', this);"><br>
+            <font class = 'Explain'>The Albedo (Color) can be used to tint the texture of the Albedo Map.</font>
+            </div>
+            
+            <div class='group'>
+            <font class='label'>Metallic:</font> <input name = 'metallic_value' readonly type = 'text' size = '5'><br> 
+            <input type="range"  style="width:100%;" min="1" max="1000" value="1" class="slider" name = 'metallic' id = 'metallic' oninput="save('metallic', this); document.materialEditor.metallic_value.value = (this.value/1000);"><br>
+            <table style='width:100%'><tr><td style='width: 15%; text-align: left;'><font class = 'Explain'>|&lt; Nonmetal</font></td><td style='width: 15%; text-align: left;'><font class = 'Explain'>| Hair</font></td><td style='width: 60%; text-align: left;'><font class = 'Explain'>| Chitin</font></td><td style='width: 10%; text-align: right;'><font class = 'Explain'>Metal &gt;|</font></td></tr></table>
+            <font class='label'>Metallic Map (Grayscale) URL:</font> <input name = 'metallicMap' type = 'text' style= 'width:100%;' oninput="save('metallicMap', this);">
+            </div>
+            
+            <div class='group'>
+            <font class='label'>Roughness:</font> <input name = 'roughness_value' readonly type = 'text' size = '5'><br>
+            <input type="range"  style="width:100%;" min="0" max="1000" value="0" class="slider" name = 'roughness' id = 'roughness' oninput="save('roughness', this); document.materialEditor.roughness_value.value = (this.value/1000);"><br>
+            <table style='width:100%'><tr><td style='width: 50%; text-align: left;'><font class = 'Explain'>|&lt; Glossy, polished, lustrous</font></td><td style='width: 50%; text-align: right;'><font class = 'Explain'>Unpolished, mat, rough &gt;|</font></td></tr></table>
+            <font class='label'>Roughness Map (Grayscale) URL:</font> <input name = 'roughnessMap' type = 'text' style= 'width:100%;' oninput="save('roughnessMap', this);">
+            </div>
+            
+            <div class='group'>
+            <font class='label'>Normal (bump) Map URL:</font> <input name = 'normalMap' type = 'text' style= 'width:100%;' oninput="save('normalMap', this);">

```

Comment:
A bump map is different from a normal map.
```suggestion
            <font class='label'>Normal Map URL:</font> <input name = 'normalMap' type = 'text' style= 'width:100%;' oninput="save('normalMap', this);">
```